packages/api-gateway/__tests__/places/getPlacesFail.test.js

```diff
@@ -0,0 +1,16 @@
+const { fastify } = require('../../src/drivers/http/server');
+
+describe('Given a client that wants to get the Places data', () => {
+  describe('When the server cannot connect to db', () => {
+    jest.mock('@booking-services/places', () => ({
```

Comment:
nit-pick: I think you can use `.mockImplementation` since you mock the whole package in `setupTests`
```suggestion
    beforeAll(() => {
      getPlaces.mockImplementation(() => new Error());
    });
```